model/project_ref.go

```diff
@@ -2245,54 +2250,78 @@ func (p *ProjectRef) GetGithubProjectConflicts() (GithubProjectConflicts, error)
 }
 
 // ValidateProjectCreation returns a boolean if you should surface the error or not.
```

Comment:
+1

Reply:
This also needs some more details about what it's validating, as a public function. And it appears that it can be expected for the inputted projectRef to be nil. Can you expand on that?

Reply:
This is also now about enabling a project, not creating one

Reply:
updated